bazel_integration_test/private/script_test.bzl

```diff
@@ -0,0 +1,116 @@
+"A facility for script tests that create workspaces and call Bazel."
+
+load(
+    "//bazel_integration_test/private:bazel_integration_test.bzl",
+    "bazel_integration_test",
+)
+
+_RUNNER_SCRIPT = """
+#!/bin/bash
+
+script="${PWD}/%s"
+
+working_dir="${BIT_WORKSPACE_DIR:-}"
+cd "${working_dir}"
+
+export BAZEL
```

Comment:
Do you still want this? I noticed that you switched to `BIT_BAZEL_BINARY`.

Reply:
No, I should remove it.

I'm fading from jet lag (just got back in the country yesterday), so I'll pick this up again in the morning and see if I can find a mac for testing. Thanks for your help, I'll let you know when I have this ready for more review.

Reply:
I think that we are there with the changes that I just pushed. I am happy to remove it and merge the PR if you would like.

Reply:
Yes, if the tests pass please remove it and merge. Thanks so much!